libs/components/datetime/src/lib/modules/datepicker/datepicker-input.directive.ts

```diff
@@ -353,16 +353,57 @@ export class SkyDatepickerInputDirective
       return;
     }
 
-    const value: any = control.value;
+    const value: Date | string = control.value;
```

Comment:
Isn't the return value of the control technically an `any`? Number, string, undefined, null, Date, etc.?

Reply:
Good catch. Switched this to `unkown`.